go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	go.mongodb.org/mongo-driver v1.4.2 // indirect
 	go.uber.org/multierr v1.6.0
 	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e // indirect
+	golang.org/x/oauth2 v0.0.0-20210402161424-2e8d93401602
```

Comment:
no need to checkin go.mod file I think.

Reply:
It's needed if the go files are committed (which is discussed elsewhere in this review)

Reply:
Keeping it for now.